BENCHMARKING.md

```diff
@@ -293,8 +293,10 @@ The supported environment variables are described in the following list:
     library formats (cuSPARSE with `cusp_` prefix or hipSPARSE with `hipsp_`
     prefix) can be used as well. Multiple options can be passed. The default is
     `csr,coo,ell,hybrid,sellp`.
-* `SOLVERS={bicgstab,bicg,cg,cgs,fcg,gmres}` - the solvers which should be
-    benchmarked. Multiple options can be passed. The default is `cg`.
+* `SOLVERS={bicgstab,bicg,cg,cgs,fcg,gmres,lower_trs,upper_trs}` - the solvers
+    which should be benchmarked. Multiple options can be passed. The default
+    is `cg`. Note that `lower/upper_trs` by default don't use a preconditioner,
+    as they are by default exact direct solvers.
```

Comment:
Not sure whether this is necessary, but should we mention that lowe/upper_trs expect a triangular matrix?

Reply:
Too late sorry, the PR is already merged 🤷‍♂️

Reply:
Essentially, I guess we should define what are 'lower_trs' and 'upper_trs', in case people are not familiar. From an applications point of view, or indeed from most common points of view, these two are really not in the same category as bicgstab, gmres, etc. We address this in some other PR.

Reply:
I don't think we have to explain everything about the TRS here necessarily, because this here is only the benchmarks description. If you want information from a user POV, the class description etc should give you all you need. On the other hand, someone who reads this should know what to benchmark, and only wants to know how.